src/utils/geo.js

```diff
@@ -66,14 +67,15 @@ export let geoCodeLocation = async (
     console.error('geoCodeLocation', data.error)
     throw new Error(data.error)
   } else {
-    var position = _.get(
-      'Response.View.0.Result.0.Location.DisplayPosition',
-      data
-    )
-
-    return {
-      latitude: position.Latitude,
-      longitude: position.Longitude,
-    }
+    return _.get('Response.View.0.Result.0', data)
   }
 }
+
+export let geoCodeLocation = async (
+  locationId,
+  hereConfig = defaultHereConfig
+) =>
+  _.get(
+    'Location.DisplayPosition',
```

Comment:
I think this is a breaking change - the prior implementation returned `{latitude, longitude}` all lowercase, but if I am reading this right, you're going to have `PascalCase` fields based on what comes back from the API

Reply:
@daedalus28 updated, ty